src/test/util/setup_common.h

```diff
@@ -52,6 +52,8 @@ struct TestOpts {
     std::vector<const char*> extra_args{};
     bool coins_db_in_memory{true};
     bool block_tree_db_in_memory{true};
+    bool setup_net{true};
```

Comment:
nit: A bit weird that this will be available as an option on all testing setups when only `TestingSetup` and it's derived classes care about it.